.github/workflows/ci.yml

```diff
@@ -80,8 +70,23 @@ jobs:
             ./crates/node/index.d.ts
           key: ${{ runner.os }}-oxide-${{ hashFiles('./crates/**/*') }}
 
-      # Nx Cloud start-ci-run
-      - run: npx nx-cloud start-ci-run --distribute-on="3 my-linux-rust-large" --stop-agents-on-failure=false --stop-agents-after="integration-test"
+      # Install Rust
+      - name: Install Rust (Stable)
+        run: rustup default stable
+
+      - name: Setup rust target
+        run: rustup target add ${{ matrix.target }}
+
+      - name: Install dependencies
+        run: pnpm install
+
+      # Nx Cloud start-ci-run using windows or linux agents accordingly
+      - run: npx nx-cloud start-ci-run --distribute-on="3 tailwind-${{ matrix.nx_ci_execution_env }}-rust" --stop-agents-on-failure=false --stop-agents-after="integration-test"
+        if: matrix.nx_ci_execution_env != 'macos'
+
+      # Nx Cloud does not support macOS agents yet, so we need to use local agents for now
+      - run: npx nx-cloud start-ci-run --stop-agents-on-failure=false --stop-agents-after="integration-test"
+        if: matrix.nx_ci_execution_env == 'macos'
```

Comment:
This sends the tasks for the windows jobs to the `tailwind-windows-rust` launch template, and the linux jobs to the linux ones.

We don't support mac in agents yet, so we'll need to run those ones inside github actions. See the `macos-dte-agents` task which spins up three mac agents below.